### PR TITLE
Issues: 8 - 11 | Fix errors in documentation

### PR DESCRIPTION
**Pull request by damrath**
Links updated, updated ResultDispatcher callback documentation and updated some erroneous parameters.

#8 #9 #10 #11

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@ build/
 node_modules/
 .idea
 *.iml
+package-lock.json
```
Comment (KrzysztofKozlowskiOBSS) [inline .gitignore]: I guess we should commit this file

Comment (damrath) [reply .gitignore]: i don't think we need it though. What do you say @toarm ?

Comment (KrzysztofKozlowskiOBSS) [reply .gitignore]: https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

Comment (damrath) [reply .gitignore]: I know they recommend doing so, however we haven't been doing this consistently, neither have we used the feature afaik. @toarm should decide

Comment (toarm) [reply .gitignore]: Honestly I never thought about package-lock.json. For me it feels initially wrong to push such a file but on the other side they are recommending it x/

I'd like to test it. If it results in merge conflicts and other annoying stuff we can remove it. Acutally I dont see any benefit in pushing it but lets stick to the recommendations for a while and see if it works out...

Comment (damrath) [reply .gitignore]: @toarm @KrzysztofKozlowskiOBSS re-added package-lock.json

**Pull request merged**